app/controllers/languages_controller.rb

```diff
@@ -0,0 +1,38 @@
+class LanguagesController < ApplicationController
+    def new
+      @language = Language.new
+    end
+    
+    # create 
+    def create
+      language = Language.new
+      language.name = params[:language][:name]
+      language.save
+      redirect_to languages_path
+    end
+    
+    # list all 
+    def index
+      @languages = Language.all
+    end
+    
+    # send back to edit
+    def edit
+      @language = Language.find(params[:id])
+    end
+    
+    # update 
+    def update
+      language = Language.find(params[:id])
+      language.name = params[:language][:name]
+      language.save
+      redirect_to languages_path      
+    end
+    
+    # delete 
+    def remove
```

Comment:
Why is this method called `remove` instead of the RESTful default name of `destroy`?